### DISCUSSION TITLE
Change text colors, with my terminal color schemes.

### DISCUSSION THREAD

**Discussion by gh0stzk** [olimorris/onedarkpro.nvim#231] [Q&A] [answered]
Hello, everything works fine, the only thing i wanna change is the text colors. I use bspwm, and i have more than 10 different themes, including my terminal color schemes, every theme change colors of everything, ( see pictures, one with dracula color scheme and one with tokyo night)

I added this to onedark plugin.

```lua
                       styles = {
				Normal = "NONE",
				Folded = "NONE",
				SignColumn = "NONE",
				Statusline = "NONE",
				Tabline = "NONE",
			},
```
As i said, works fine, my background changes with my terminal color, etc etc.
But the text didnt change, is always onedark colors. What group i need to add to make the text changes???
Thank you!!

![Shot-2024-03-02-084156](https://github.com/olimorris/onedarkpro.nvim/assets/67278339/c6c73d37-1a27-4ff1-98f6-32d154ff7375)
![Shot-2024-03-02-084214](https://github.com/olimorris/onedarkpro.nvim/assets/67278339/04735f12-22e8-4a5d-9e0c-1dd8dacb0c5e)

**Comment by olimorris** [comment, marked as answer]
How are you changing the colorscheme within Neovim? If I run `:colorscheme onedark_vivid` or `:colorscheme onelight` I can see the theme change
